app/src/main/res/values/strings.xml

```diff
@@ -927,6 +927,8 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_construction_steps_title">Are these steps completed?</string>
     <string name="quest_construction_generic_title">Is this path completed?</string>
 
+    <string name="quest_crossing_kerb_height_title">What’s the height of the curbs of this crossing?</string>
```

Comment:
Maybe this is a better wording? @arrival-spring 

```suggestion
    <string name="quest_crossing_kerb_height_title">What’s the height of the curbs at this crossing?</string>
```

Reply:
"at" is better than "of", but because curbs is plural, it should be "What are the heights of the curbs at this crossing?"

Reply:
But it is asking for a single height, i.e. the user should be not sure what to answer if the height**s** differ (-> leave note)

Reply:
Yes, I see. Flo's suggestion is fine, height is singular, so "what's" matches